common/lib/xmodule/xmodule/templates/html/current_event.yaml

```diff
@@ -0,0 +1,11 @@
+---
+metadata:
+    display_name: Current Event
+data: |
+      <div class="div-currentevent">
+        <img class="icon-image" src="/static/in_the_news.png" alt="">
+        <div>
+          <h4>Current Event</h4>
+      		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```

Comment:
Nit: I would break those long lines to make the file more polite. But can be ignored.